proxy_handler_test.cpp

```diff
@@ -0,0 +1,43 @@
+#include "gtest/gtest.h"
+#include "request.hpp"
+#include "response.hpp"
+#include "proxy_handler.hpp"
+#include "request_handler.hpp"
+#include "config_parser.h"
+#include <string>
+#include <limits.h>
+#include <unistd.h>
+namespace http {
+namespace server {
+
+TEST(ProxyHandlerTest, InitTest) {
+    ProxyHandler proxy_handler;
+	NginxConfig* config = new NginxConfig();
+    NginxConfigStatement* st = new NginxConfigStatement();
+
+    st->tokens_.push_back("root");
+    st->tokens_.push_back("ucla.edu");
+
```

Comment:
I see that you use the tokens_.push_back() in both tests. Maybe you could create a test fixture, so that you could do it only once.